foxy.cmake

```diff
@@ -1,3 +1,9 @@
 set(FOXY_TESTING ON)
+set(Boost_USE_STATIC_LIBS ON)
+set(Boost_USE_DEBUG_LIBS ON)
+set(Boost_USE_RELEASE_LIBS OFF)
+set(Boost_USE_DEBUG_RUNTIME ON)
+set(BOOST_ROOT "/Users/cmaza/cpp/boost-169/install")
```

Comment:
You might want `BOOST_ASIO_NO_DEPRECATED`

Reply:
I wound up adding this directly to the core lib instead of a toolchain file (as I believe it belongs in Foxy's core).


Marking as resolved.